recipes/cppyythonizations/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "cppyythonizations" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/flatsurf/cppyythonizations/releases/download/{{ version }}/cppyythonizations-{{ version }}.tar.gz
+  sha256: 94cf064dc6c5f048b5987fb9dab295700403255ab993ff9b29cda9f3a9629517
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  build:
+    - automake
+    - coreutils
+    - make
+  host:
+    - python >=3.6
+    - setuptools
+  run:
+    - python >=3.6
+    - cppyy
+
+test:
+  imports:
+    - cppyythonizations
```

Comment:
It looks like this `import` basically does nothing. Can you add an import test that actually triggers a compile? (and really, more imports or tests that validate things are working.) I'm pretty sure we're not creating an environment where everything works yet.

Reply:
Ok. Let me add more imports then.